src/java.instrument/share/classes/sun/instrument/InstrumentationImpl.java

```diff
@@ -485,13 +480,8 @@ private TransformerManager findTransformerManager(ClassFileTransformer transform
 
     // Enable or disable Java programming language access checks on a
     // reflected object (for example, a method)
-    @SuppressWarnings("removal")
     private static void setAccessible(final AccessibleObject ao, final boolean accessible) {
-        AccessController.doPrivileged(new PrivilegedAction<Object>() {
-                public Object run() {
-                    ao.setAccessible(accessible);
-                    return null;
-                }});
+        ao.setAccessible(accessible);
```

Comment:
I think you can remove this method, and instead change the usage in loadClassAndStartAgent to just call setAccessible.

Reply:
will do, yes it is trivial now.